src/components/OpeningCrawl.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './OpeningCrawl.css';
+
+function OpeningCrawl(props) {
```

Comment:
You can destructure the props in the function signature, so it will be easier to read all the props that are used by a component.

Reply:
OpeningCrawl.js now uses destructured props.